index.html

```diff
@@ -50,6 +50,110 @@ <h1 >Hey There.
+                        <img src="https://picsum.photos/200/300" alt="">
+                    </div>
+                    <div class="content">
+                        <h2>Lorem ipsum</h2>
+                        <p>
+                            Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolorum necessitatibus
+                            voluptatum et facere illum aliquam nulla distinctio quidem nostrum quas iusto, 
+                            obcaecati sequi, aliquid quos at reprehenderit deleniti eius. Necessitatibus.
+                        </p>
+                        <ul>
+                            <li>html</li>
+                            <li>css</li>
+                            <li>javascript</li>
+                        </ul>
+                    </div>
+                    <button class="live">See Project</button>
+                </article>
+                <article class="card work">
+                    <div class="background">
+                        <img src="https://picsum.photos/200/300" alt="">
+                    </div>
+                    <div class="content">
+                        <h2>Lorem ipsum</h2>
+                        <p>
+                            Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolorum necessitatibus
+                            voluptatum et facere illum aliquam nulla distinctio quidem nostrum quas iusto, 
+                            obcaecati sequi, aliquid quos at reprehenderit deleniti eius. Necessitatibus.
+                        </p>
+                        <ul>
+                            <li>html</li>
+                            <li>css</li>
+                            <li>javascript</li>
+                        </ul>
+                    </div>
+                    <button class="live">See Project</button>
+                </article>
+                <article class="card work">
+                    <div class="background">
+                        <img src="https://picsum.photos/200/300" alt="">
+                    </div>
+                    <div class="content">
+                        <h2>Lorem ipsum</h2>
+                        <p>
+                            Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolorum necessitatibus
+                            voluptatum et facere illum aliquam nulla distinctio quidem nostrum quas iusto, 
+                            obcaecati sequi, aliquid quos at reprehenderit deleniti eius. Necessitatibus.
+                        </p>
+                        <ul>
+                            <li>html</li>
+                            <li>css</li>
+                            <li>javascript</li>
+                        </ul>
+                    </div>
+                    <button class="live">See Project</button>
+                </article>
+            </div>
+        </section>
+        <section class="about">
+            <div class="fl-box bg-ilust">
+                <h1>About me</h1>
```

Comment:
[OPTIONAL]
- The title should be placed on the right. Kindly fix it 😃 See the screenshot below:

<img width="224" alt="image" src="https://user-images.githubusercontent.com/77190203/148592273-1c195fe3-d362-484e-bbb1-ab94824a0e95.png">